src/cdk/scrolling/scrollable.ts

```diff
@@ -20,36 +21,32 @@ import {ScrollDispatcher} from './scroll-dispatcher';
   selector: '[cdk-scrollable], [cdkScrollable]'
 })
 export class CdkScrollable implements OnInit, OnDestroy {
-  private _elementScrolled: Subject<Event> = new Subject();
-  private _scrollListener = (event: Event) => this._elementScrolled.next(event);
+  private _destroyed = new Subject();
+
+  private _elementScrolled: Observable<Event> = Observable.create(observer =>
```

Comment:
Why do you need the `Observable.create` here? You can just do this:

```ts
private _elementScrolled = this._ngZone.runOutsideAngular(() => {
  return fromEvent(...).pipe(takeUntil(this._destroyed), tap(() => ...));
});
```

Reply:
What does the `tap(() => ...)` do here? 

I was under the impression that the call to `subscribe` has to occur inside the `runOutsideAngular` function, that's why I did it this way

Reply:
Sorry, I forgot that about having to subscribe.